mainnet-contracts/src/interface/IWithdrawalManager.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.8.0 <0.9.0;
+
+import { PufferVaultV3 } from "../PufferVaultV3.sol";
+
+/**
+ * @title IWithdrawalManager
+ * @author Puffer Finance
+ * @notice Interface for the WithdrawalManager contract
+ * @custom:security-contact security@puffer.fi
+ */
+interface IWithdrawalManager {
+    /// @notice Thrown when attempting to finalize a batch that has already been finalized
```

Comment:
The rest of our code base uses multi line natspec /** ...
Even this interface has it from lines 6-11. We should stick to one code style, even though it doesn't make any real difference